dace/codegen/targets/rtl.py

```diff
@@ -0,0 +1,544 @@
+# Copyright 2019-2020 ETH Zurich and the DaCe authors. All rights reserved.
+
+import itertools
+
+from typing import List, Tuple, Dict
+
+from dace import registry, symbolic, nodes, StorageType, Language, SDFG
+from dace.config import Config
+from dace.codegen.targets.target import TargetCodeGenerator
+from dace.codegen.prettycode import CodeIOStream
+from dace.codegen.targets.framecode import DaCeCodeGenerator
+from dace.codegen.dispatcher import TargetDispatcher
+from dace.sdfg.graph import MultiConnectorEdge
+from dace.sdfg.state import StateSubgraphView
+from dace.sdfg import find_input_arraynode, find_output_arraynode
+from dace.codegen.codeobject import CodeObject
+from dace.dtypes import pointer, vector
```

Comment:
Style: please import package names rather than classes (exceptions: `typing`, transformations, the `SDFG` and `SDFGState` classes)